core/src/main/java/io/aiven/kafka/tieredstorage/chunkmanager/cache/ChunkCacheConfig.java

```diff
@@ -33,9 +33,9 @@ public class ChunkCacheConfig extends AbstractConfig {
             + "where \"-1\" represents infinite retention";
     private static final long DEFAULT_CACHE_RETENTION_MS = 600_000;
 
-    private static final String CACHE_PREFETCHING_SIZE_CONFIG = "prefetching.size";
+    private static final String CACHE_PREFETCHING_SIZE_CONFIG = "prefetching.bytes";
```

Comment:
We are using `.size` for most bytes configurations. Also, final size is not specifically this value but whatever chunks can make up to this size, right? Finally, `ing` suffix doesn't add much so we may as well drop it:
```suggestion
    private static final String CACHE_PREFETCH_MAX_SIZE_CONFIG = "prefetch.max.size";
```

Reply:
It's not max size but rather min, since we are prefetching the whole chunk if requested range ends or starts in the middle of the chunk.

Reply:
Oh actually you are right, it's max size

Reply:
Fixed